src/view/components/typography/body.styl

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// Body classes are not bundled with React-Spectrum, so we have
+// to import them directly.
+@import "@spectrum-css/typography/dist/index-vars.css";
```

Comment:
Technically, not having this file was a bug. If the Body component were used, the Body spectrum-css files wouldn't be included and therefore the `spectrum-Body` class (and other associated Body classes) wouldn't be included or applied. However, we're only using Body in one place at the moment, and it's used on a view that also uses Heading, and Heading also imports this same `index-vars.css` file, which is why we weren't/aren't seeing issues.

This actually shows a bit of a weakness in the Heading and Body tests that are included in this PR, because none of these new tests are testing that the computed styles the element is receiving are coming from the `index-vars.css` file; we're just testing that we are setting the right classes and styles on the element.

One thing we _could_ do is pick a specific style that should _only_ be coming from `index-vars.css` and test that the element ends up with a computed style that matches. At that point, though, we're kind of testing internals of `index-vars.css` (if style values inside index-vars.css change, our test may break) and also assuming that the computed style will continue being uniquely sourced from `index-vars.css` versus something else in React-Spectrum.

I'm okay with just testing that we're setting the right class and styles on the element as I've done in this PR and dealing with the risk trade-off, but I'm also open to other opinions.

Reply:
Having a Body component isolated test would find such problems, but that may be out of scope for this PR.

Reply:
I think we may be miscommunicating and it's probably my fault. This PR does have a test that isolates the Body component, if I understand what you mean. The test, though, asserts that we apply the right classes to the element, not that the computed font size/style/color is a specific value being provided by `index-vars.css`. In other words, the test would still pass even if `index-vars.css` weren't loaded by Body.